### DISCUSSION TITLE
Adding a folder as a menu is gone with Windows 11 24H2

### DISCUSSION THREAD

**Discussion by prjahelka** [valinet/ExplorerPatcher#3310] [Q&A]
Is this problem related to the dropping of the Windows 10 Taskbar style?

I used to see Create Shortcut in the Task Manager context menu - not there anymore.
I had two folders pinned to my taskbar. I exported ExplorerPatcher settings before installing 22H4, 
then uninstalled EP so that the upgrade would run.
After the upgrade and reinstalling EP and importing my settings, I don't see the shortcut menus.

**Comment by pyrates999** [comment]
Pinning folders to the taskbar is not a feature that EP provides.
